server/lib/validation/languages/lo.js

```diff
@@ -0,0 +1,34 @@
+// Lao rules
+// use any rule from Thai rules https://github.com/common-voice/sentence-collector/blob/main/server/lib/validation/languages/th.js
+const MIN_LENGTH = 2;
+const MAX_LENGTH = 100;
+
+const INVALIDATIONS = [{
+  fn: (sentence) => {
+    return sentence.length < MIN_LENGTH || sentence.length > MAX_LENGTH;
+  },
+  error: `Number of characters must be between ${MIN_LENGTH} and ${MAX_LENGTH} (inclusive)`,
```

Comment:
As with other languages, I think it would make sense to translate these error messages into Lao. What do you think?

Reply:
> As with other languages, I think it would make sense to translate these error messages into Lao. What do you think?

I agree with you. I will translate these messages into Lao.